From ec12d646175e938f094de071d21a64cf6d1fe8cc Mon Sep 17 00:00:00 2001
From: Mohammad Athari Bin Ismail <mohammad.athari.ismail@intel.com>
Date: Fri, 20 Nov 2020 17:41:02 +0800
Subject: [PATCH] net: phy: Updated PHY ID mask for GPY PHY

Updated PHY ID mask for GPY PHY to 0xfffffc00. Only OUI field is needed
without device number and device revision number.

Signed-off-by: Mohammad Athari Bin Ismail <mohammad.athari.ismail@intel.com>
---
 include/linux/intel_phy.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/include/linux/intel_phy.h b/include/linux/intel_phy.h
index 31a7dfdb50b3..4d33c453cf91 100644
--- a/include/linux/intel_phy.h
+++ b/include/linux/intel_phy.h
@@ -3,7 +3,7 @@
 #define _INTEL_PHY_H
 
 /* Mask used for ID comparisons */
-#define INTEL_PHY_ID_MASK		0xffffffff
+#define INTEL_PHY_ID_MASK		GENMASK(31, 10)
 
 /* GPY PHY IDs */
 #define INTEL_PHY_ID_GPY		0x67C9DC00
-- 
2.17.1

